gluecodium/src/test/resources/smoke/attributes/input/Attributes.lime

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2016-2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+package smoke
+
+@Java(Attribute = "OnClass")
```

Comment:
May be better to keep full form like `Attribute = "@OnClass" `? For example in C++ it looks like `[[attribute]]`, so may be we want to make it more explicit?

Reply:
I really don't like the idea of just taking a string value "foobar" from `Attribute="foobar"` verbatim. This opens a wide door to code injection hacks from "smart" IDL authors.

Admittedly, even in the current form someone can attempt code injection, so I'll add a few guards against this right now.